src/resources/anim-state-graph.js

```diff
@@ -1,6 +1,125 @@
 import { http, Http } from '../net/http.js';
 
-import { AnimStateGraph } from '../framework/components/anim/state-graph.js';
+/**
+ * @private
+ * @class
+ * @name pc.AnimStateGraph
+ * @classdesc Creates an AnimStateGraph asset resource from a blob of JSON data that represents an anim state graph.
+ */
+class AnimStateGraph {
```

Comment:
I would put this in `src/anim/state-graph/anim-state-graph.js`.

Reply:
👍 Sure it can go there